src/main/java/pl/javastart/task/PalindromeService.java

```diff
@@ -1,9 +1,17 @@
 package pl.javastart.task;
 
-public class PalindromeService {
+import java.util.Objects;
 
-    // nie zmieniaj sygnatury tej metody. Jest ona testowana w PalindromeServiceTest
+public class PalindromeService {
     public boolean isPalindrome(int[] array) {
-        return false;
+        if (Objects.isNull(array) || array.length == 0) {
+            return false;
+        }
+        for (int i = 0, j = array.length - 1; i < array.length; i++, j--) {
```

Comment:
Jeżeli masz tablicę np. o rozmiarze 10, to wystarczy, że ta pętla wykona się 5 razy, a nie 10, bo porównanie i=4 i j=6 elementu to, to samo co porównanie i=6 i j=4 elementu